File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-05/plagiarized/L2/07/Kasus5L2.kt
package IR

import java.util.Scanner

fun main() {
    //input data
    print("Enter an integer: ")
    val inp = Scanner(System.`in`)
    val angka = inp.nextInt()
    reverseNumber(angka)
}

fun reverseNumber(angka: Int) {
    //proses membalikkan angka
    var num = angka
    while (num != 0) {
        //mengambil angka terakhir
        val temp = num % 10
        //tampilkan angka yang diambil
        print(temp)
        //membuang angka terakhir
        num /= 10
    }
    println()
}

File: datasets/IR-Plag/convertai-original-java-rest-python/case-05/plagiarized/L2/07/Kasus5L2.py
import sys

def reverse_number(angka):
    #proses membalikkan angka
    while angka != 0:
        #mengambil angka terakhir
        temp = angka % 10
        #tampilkan angka yang diambil
        print(temp, end='')
        #membuang angka terakhir
        angka = angka // 10
    print()

if __name__ == "__main__":
    #input data
    angka = int(input("Enter an integer: "))
    reverse_number(angka)

